hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -56,8 +59,3 @@ After five to ten seconds, the Raspberry Pi should begin booting and show the ra
 
 - The default timeout for checking bootable USB devices is 2 seconds. Some flash drives and hard disks power up too slowly. It is possible to extend this timeout to five seconds (add a new file `timeout` to the SD card), but note that some devices take even longer to respond.
 - Some flash drives have a very specific protocol requirement that is not handled by the bootcode and may thus be incompatible.
-
-<a name="pi4"></a>
```

Comment:
The existence of this anchor suggests that something is probably linking to it, so it ought to be kept? :shrug:

Reply:
An anchor without text is not useful, the browser behaviour would be to go to the top of the page instead which is actually what we want here

Reply:
I meant you could add it back to just above the (now moved) "## Raspberry Pi 4" header. While they look weird, empty anchors are unfortunately what's needed to get anchors working on the raspberrypi.org site, see e.g. https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/msd.md#pi4